sklearn/ensemble/_voting.py

```diff
@@ -441,3 +460,8 @@ def transform(self, X):
         """
         check_is_fitted(self)
         return self._predict(X)
+
+    def _log_message(self, name, idx, total):
```

Comment:
Since the function is called in both `Voting*` estimator, we could move it in `_BaseVoting` avoiding to have repeated code in both `Voting*` classes?